resources/victron-virtual-functions.js

```diff
@@ -74,61 +126,268 @@
         { id: 'unit', type: 'text', placeholder: 'Unit', title: 'Unit (center)', style: 'width:120px;' }
       ]
     },
-    9: {
+    [victronVirtualConstantsExports.SWITCH_TYPE_MAP.THREE_STATE]: {
       label: 'Three-state switch',
       fields: [...COMMON_SWITCH_FIELDS]
     },
-    10: {
+    [victronVirtualConstantsExports.SWITCH_TYPE_MAP.BILGE_PUMP]: {
       label: 'Bilge pump control',
       fields: [...COMMON_SWITCH_FIELDS]
     }
   };
 
+  const SWITCH_TYPE_DOCS = {
+    [victronVirtualConstantsExports.SWITCH_TYPE_MAP.MOMENTARY]: {
+      text: `
+      <div>
+        <strong>Most relevant path(s):</strong>
```

Comment:
Why have a section `Most relevant path(s)`? Analogous to `Outputs`, we could list `Inputs`.

Related: Is there a place where I can see which properties are writeable, and which ones are read-only? When testing, I'm just guessing what might be writable.

It may also help to document here how to set the inputs: The simplest I can come up with is a function node with:

```javascript
msg.payload = {
    'SwitchableOutput/output_1/State': 1
};
return msg;
```

Maybe there is an easier way?

Reply:
I opted to have most relevant paths as some of the rest of the paths (as described on https://github.com/victronenergy/venus/wiki/dbus#switch) are not that useful to modify during runtime. 

But we could refer to the full documentation. I suggest awaiting user feedback on this.